### ISSUE TITLE
Cannot be used with Unity Netcode package

### ISSUE THREAD

**Issue by BentiGorlich** [Hello-Meow/ModTool#10]
I love this tool and it is just very simple to use, bus as soon as I import both, the Netcode Package and the ModTool, Unity goes crazy and doesn't include the UnityEngine.UI package anymore and I get the message that the Mono.Cecil library is included multiple times. When I delete the Mono.Cecil.dll inside of the ModTool package, then I get the error that the dll is missing, although it is included via NuGet in my C# package...

TL;DR the ModTool doesn't work with the Netcode package from Unity

**Comment by Hello-Meow** [comment]
[This is a known issue.](https://github.com/Hello-Meow/ModTool/issues/7) ModTool and some Unity packages use a different version of Mono.Cecil.

**Comment by BentiGorlich** [comment]
Ok, I apologize for any inconvinience.
